BLSQ/iaso
# IA-1697: refactor InstancePopUpComponent as function

Old class component that need to be refactored in order to allow upgrading leaflet

Related JIRA tickets : IA-1697

## Self proof reading checklist


## Changes

- replaced HOC with hooks for
    - redux state selectors
    - styling
    - translations
- wrapped callbacks in `useCallback`
- wrapped values computed in `render` with `useMemo` when necessary
- Fixed a styling bug with the "See" button, where only the link was actually clickable
- Use the new component in `InstancesMap` and `OrgUnitMapComponent`

## How to test

Log in, go to Instances > map tab, click on an instance to see the pop up. Click the "See" button: you should be redirected even if you don't click on the text

Go to Org units and do the same

Apart from the link, nothing should have changed

## Print screen / video

https://user-images.githubusercontent.com/38907762/214267904-f8152e00-dfac-42bf-892d-5badbb0490cf.mov

## Notes
merges into https://github.com/BLSQ/iaso/pull/328
